config/jobs/kubernetes/sig-cloud-provider/azure/release-1.23.yaml

```diff
@@ -17,7 +17,7 @@ presubmits:
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: release-1.7
+        base_ref: release-1.8
```

Comment:
A recent test run after this change failed:

- https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-capz-azure-disk/1634192105758789632

@andyzhangx do you think the above test failure has anything to do w/ CAPZ ci-entrypoint.sh?

cc @CecileRobertMichon